.changeset/@mastra_deployer-9780-dependencies.md

```
@@ -0,0 +1,5 @@
+---
+"@mastra/deployer": patch
+---
+dependencies updates:
+  - Updated dependency [`@rollup/plugin-commonjs@29.0.0` ↗︎](https://www.npmjs.com/package/@rollup/plugin-commonjs/v/29.0.0) (from `28.0.6`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)